lib/winston-graylog2.js

```diff
@@ -80,12 +80,9 @@ let Graylog2 = (winston.transports.Graylog2 = function(options) {
   this.name = options.name || 'graylog2';
   this.silent = options.silent || false;
   this.json = options.json || false;
-  this.colorize = options.colorize || false;
```

Comment:
why are we removing these?

Reply:
Becoz Graylog does not support colorize and prettyprint.